Translators/Incident/sample.env

```diff
@@ -0,0 +1,10 @@
+dual_carriageway_endpoint=https://dtdapps.coloradodot.info/arcgis/rest/services/LRS/Routes_withDEC/MapServer/exts/CdotLrsAccessRounded
+DB_HOST=
+DB_NAME=
+DB_PASS=
+DB_USER=
+LOGGING_LEVEL=INFO
+CDOT_FEED_ENDPOINT=
+CDOT_FEED_API_KEY=
+ODE_ENDPOINT=

```

Comment:
Let's tweak this to TIM_MANAGER_ENDPOINT since it no longer is submitting to the ODE

Reply:
Updated!